upipe_avcodec_encode unconditionally uses bitstream

```
/home/james/src/upipe/lib/upipe-av/upipe_avcodec_encode.c:80:10: fatal error: bitstream/mpeg/h264.h: No such file or directory
   80 | #include <bitstream/mpeg/h264.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~
```

--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -97,8 +97,10 @@ noinst_PROGRAMS += grid
 endif
 endif
 
+if HAVE_BITSTREAM
 if HAVE_DLFCN_H
 noinst_PROGRAMS += blackmagic
+endif
 endif
 endif # swresample
 

--- a/lib/upipe-av/Makefile.am
+++ b/lib/upipe-av/Makefile.am
@@ -10,7 +10,6 @@ libupipe_av_la_SOURCES = \
 	upipe_av_codecs.c \
 	upipe_avformat_sink.c \
 	upipe_avformat_source.c \
-	upipe_avcodec_encode.c \
 	upipe_av_codecs.pl \
 	avcodec_include.h \
 	ubuf_av.c \
@@ -24,7 +23,7 @@ libupipe_av_la_LIBADD = $(top_builddir)/lib/upipe/libupipe.la \
 libupipe_av_la_LDFLAGS = -no-undefined
 
 if HAVE_BITSTREAM
-libupipe_av_la_SOURCES += upipe_avcodec_decode.c
+libupipe_av_la_SOURCES += upipe_avcodec_decode.c upipe_avcodec_encode.c
 libupipe_av_la_CFLAGS += $(BITSTREAM_CFLAGS)
 endif
 

--- a/tests/checkasm/Makefile.am
+++ b/tests/checkasm/Makefile.am
@@ -21,8 +21,6 @@ checkasm_LDADD += \
     $(top_builddir)/lib/upipe-hbrmt/libupipe_hbrmt_la-sdidec.o \
     $(top_builddir)/lib/upipe-hbrmt/libupipe_hbrmt_la-sdienc.o \
     $(NULL)
-
-checkasm_CPPFLAGS += -DHAVE_SDI
 endif
 
 if HAVE_X86ASM

--- a/tests/checkasm/sdi_input.c
+++ b/tests/checkasm/sdi_input.c
@@ -39,10 +39,13 @@ void checkasm_check_sdi_input(void)
     struct {
         void (*uyvy)(const uint8_t *src, uint16_t *dst, uintptr_t pixels);
     } s = {
+#ifdef HAVE_BITSTREAM_COMMON_H
         .uyvy = upipe_sdi_to_uyvy_c,
+#endif
     };
 
 #ifdef HAVE_X86ASM
+#ifdef HAVE_BITSTREAM_COMMON_H
     int cpu_flags = av_get_cpu_flags();
 
     if (cpu_flags & AV_CPU_FLAG_SSSE3) {
@@ -51,6 +54,7 @@ void checkasm_check_sdi_input(void)
     if (cpu_flags & AV_CPU_FLAG_AVX2) {
         s.uyvy = upipe_sdi_to_uyvy_avx2;
     }
+#endif
 #endif
 
     if (check_func(s.uyvy, "sdi_to_uyvy")) {

--- a/tests/checkasm/uyvy_input.c
+++ b/tests/checkasm/uyvy_input.c
@@ -40,10 +40,13 @@ void checkasm_check_uyvy_input(void)
     struct {
         void (*sdi)(uint8_t *dst, const uint8_t *src, uintptr_t pixels);
     } s = {
+#ifdef HAVE_BITSTREAM_COMMON_H
         .sdi = upipe_uyvy_to_sdi_c,
+#endif
     };
 
 #ifdef HAVE_X86ASM
+#ifdef HAVE_BITSTREAM_COMMON_H
     int cpu_flags = av_get_cpu_flags();
 
     if (cpu_flags & AV_CPU_FLAG_SSSE3) {
@@ -55,6 +58,7 @@ void checkasm_check_uyvy_input(void)
     if (cpu_flags & AV_CPU_FLAG_AVX2) {
         s.sdi = upipe_uyvy_to_sdi_avx2;
     }
+#endif
 #endif
 
     if (check_func(s.sdi, "uyvy_to_sdi")) {

